pandas/tests/io/test_spss.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas.util import testing as tm
+
+from pandas.io.spss import read_spss
+
+pyreadstat = pytest.importorskip("pyreadstat")
+
+
+def test_spss_labelled_num(datapath):
+    # test file from the Haven project (https://haven.tidyverse.org/)
+    fname = datapath("io", "data", "labelled-num.sav")
+
+    df = read_spss(fname, categorical=True)
```

Comment:
style nitpick: can you use `pd.read_spss` here? That'll better test the user-facing behavior.

Reply:
Done.